manticore/core/cpu/x86.py

```diff
@@ -5020,12 +5020,15 @@ def _pcmpxstrx_polarity(self, res1, ctlbyte, arg2len):
         stepsize = self._pcmpxstrx_srcdat_format(ctlbyte)
         if (Operators.EXTRACT(ctlbyte, 4, 2) == 0):
             res2 = res1
+            pass
         if (Operators.EXTRACT(ctlbyte, 4, 2) == 1):
             res2 = ((1 << (128 / stepsize)) - 1) ^ res1
         if (Operators.EXTRACT(ctlbyte, 4, 2) == 2):
             res2 = res1
+            pass
         if (Operators.EXTRACT(ctlbyte, 4, 2) == 3):
             res2 = ((1 << arg2len) - 1) ^ res1
+            pass
```

Comment:
The `pass` here are redundant.

Reply:
I didn't add that pass, it was there originally. autoflakes found a lot of redundant `pass` statements, which I removed and then promptly broke all our tests :-x. Safer to just leave these for now.